vespalib/src/vespa/vespalib/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ vespa_add_library(vespalib
     INSTALL lib64
     DEPENDS
     ${VESPA_GCC_LIB}
+    ssl
+    crypto
```

Comment:
Probably it would be better to use ```$(OPENSSL_LIBRARIES}``` here and execute ```find_package(OpenSSL)``` above this statement.

Reply:
This is probably an issue somewhere else in rhel 6 build environment.

Reply:
vespa_add_target_package_dependency(vespalib OpenSSL) should have created the proper dependency.

Reply:
Closing